scripts/metavar.py

```diff
@@ -1209,9 +1211,10 @@ def __init__(self, prop_dict, source, run_env, context=None,
                 del prop_dict[prop.name]
             # end if
         # end for
-        # Initialize Var
+        # Initialize Var; skip the parse checkers on the Fortran side since the
+        #  checks are already done during metadata parsing
         super().__init__(prop_dict, source, run_env, context=context,
-                         clone_source=clone_source)
+                         clone_source=clone_source, skip_checks=True)
```

Comment:
This feels like the wrong place to add this check as the problem (which I do not really follow without a regression test to read) is not within this type but its use in parse_fortran.py.
I suggest adding `skip_checks` as an input to the `__init__` method for this class and calling it from parse_fortran.py